src/frontend/src/drakulaAi2nd.js

```diff
@@ -0,0 +1,153 @@
+import { getGlobals, updateGlobals } from './testMainGameLoop.js';
```

Comment:
I assume since `drakulaAi3rd` exists we can just use that and delete these two

Reply:
the main thing is drakulaAi2nd.js